test.py

```diff
@@ -108,6 +108,31 @@ def test_create_app(self):
         self.assertIn("app3", self.qiwis._dockWidgets)
         self.assertIn("app3", self.qiwis._subscribers["ch1"])
 
+    @mock.patch("qiwis.Qiwis.destroyApp")
+    def test_create_existing_app(self, mockedDestroyApp):
+        """Tests for the case where trying to create an existing app."""
+        orgApp = self.qiwis._apps["app2"]
+        app_ = mock.MagicMock()
+        app_.cls = "cls2"
+        app_.frames.return_value = (QWidget(),)
+        cls = mock.MagicMock(return_value=app_)
+        setattr(self.mocked_import_module.return_value, "cls2", cls)
+        # The original app will not be replaced.
+        self.qiwis.createApp(
+            "app2",
+            qiwis.AppInfo(**{"module": "module2", "cls": "cls2"})
```

Comment:
Why don't you write: `qiwis.AppInfo(module="module2", cls="cls2")`?

Reply:
Oh, it looks clear!
I will apply this to other parts, too.